apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedModal/anonymous-links-tab.tsx

```
@@ -0,0 +1,346 @@
+
+  const handleGenerateLinks = async (count: number) => {
+    try {
+      const response = await generateSingleUseIdsAction({
+        surveyId: survey.id,
+        isEncrypted: singleUseEncryption,
+        count,
+      });
+
+      const baseSurveyUrl = getSurveyUrl(survey, publicDomain, "default");
+
+      if (!!response?.data?.length) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)